tutorials/monitoring/setup/index.md

```diff
@@ -0,0 +1,208 @@
+---
+title: "Monitoring NServiceBus solutions: Getting started"
+reviewed: 2017-11-07
+summary: Setting up the Particular Service Platform to monitor an NServiceBus system.
+component: Core
+versions: '[5,)'
+---
+
+This tutorial will guide you through the steps required to set up your NServiceBus solution for monitoring with the Particular Service Platform.
+
+
+## Component overview
+
+```mermaid
+graph TD
+
+subgraph Endpoint
+  nsb(NServiceBus) --> metrics
+  metrics(NServiceBus.Metrics) --> plugin
+
+  plugin(NServiceBus.Metrics.ServiceControl)
+end
+
+plugin --> q(Monitoring Queue)
+
+nsb --> auditQ(Audit Queue)
+nsb --> errorQ(Error Queue)
+
+subgraph ServiceControl
+  scinstance
+  scmonitoring
+end
+
+auditQ --> scinstance(ServiceControl Instance)
+errorQ --> scinstance
+
+q --> scmonitoring[Monitoring Instance]
+
+scmonitoring --> pulse[ServicePulse]
+scinstance --> pulse
+
+```
+
+![the above renders like this](component-overview.png)
+
+Every system being monitored will push data to three main queues:
+
+- Monitoring: Each endpoint will gather and send metric data to this queue. This data holds information about the health and performance of an endpoint.
+- Audit: When an endpoint successfully processes a message, a copy of the message will be forwarded to this queue. The audited message will have additional headers which contain data about the processing of the message.
+- Error: When an endpoint fails to process a message (after exhausting automated retry attempts), a copy of the message is fowarded to this queue. The failed message will have additional headers which contain data about the failed attempt to process the message.
+
+The core NServiceBus package can be configured to send messages to the Audit and Error queues. In order to send data to the Monitoring queue, two additional packages are required:
```

Comment:
> The core NServiceBus package can be configured

I'd say we configure the solution using NServiceBus, rather than configure the package, that may be a bit confusing. Maybe "Any solution using NServiceBus (i.e. referencing `NServiceBus` package) can be..."?